src/modules/transaction/utils/moduleCommand.js

```diff
@@ -0,0 +1,35 @@
+import { MODULE_COMMANDS_NAME_MAP } from 'src/modules/transaction/configuration/moduleCommand';
+
+// TODO: Fix the mapping based on new id changes
+// We will address these problem in issue https://github.com/LiskHQ/lisk-desktop/issues/4400
+
+const getModuleCommandSenderLabel = (t = str => str) => ({
```

Comment:
this `t` mapping could be much more straight forward using the `i18next.t()` singleton :)

Reply:
with this, we can just do:

```js
// ...
import i18next from "i18next"

const getModuleCommandSenderLabel = ({
  [MODULE_COMMANDS_NAME_MAP.transfer]: i18next.t('Sender'),
  // ...
})
// ...

```

Reply:
For now we can leave it as it is, my feedback on this implementation would be when we refactor the multistep component.

Each Command should have props `name, title` so we don't need this logic. Then essentially Each command and params will have its name and its title.

Reply:
ok